swagger.yaml

```diff
@@ -1573,3 +1592,162 @@ definitions:
+              type: string
+              description: "The title of a methodology"
+      national_statistic:
+        type: boolean
+        description: "The flag indicating the resource is a national statistic. These are certified as compliant with the Code of Practice for Official Statistics"
+      next_release:
+        type: string
+        description: "The next release date for a dataset"
+      publications:
+        type: array
+        description: "A list of publications for a dataset"
+        items:
+          type: object
+          properties:
+            description:
+              type: string
+              description: "The description of a publication"
+            href:
+              type: string
+              description: "The url to a publication"
+            title:
+              type: string
+              description: "The title of a publication"
+      publisher:
+        $ref: '#/definitions/Publisher'
+      qmi:
+        description: "Object containing information on the quality and methodology index of a dataset"
+        type: object
+        properties:
+          description:
+            type: string
+            description: "The description of a quality and methodology index"
+          href:
+            type: string
+            description: "The url to a quality and methodology index"
+          title:
+            type: string
+            description: "The title of a quality and methodology index"
+      related_datasets:
+        type: array
+        description: "A list of objects containing information of datasets related to a dataset"
+        items:
+          type: object
+          properties:
+            href:
+              type: string
+              description: "The url to a related dataset"
+            title:
+              type: string
+              description: "The title of a related dataset"
+      release_date:
+        description: "The release date of this version of the dataset"
+        type: string
+      release_frequency:
+        type: string
+        description: "The release frequency of a dataset"
+      temporal:
+        $ref: '#/definitions/Temporal'
+      theme:
+        type: string
+        description: "The theme for a dataset"
+      title:
+        type: string
+        description: "The title of the dataset"
+        example: "CPI"
+      unit_of_measure:
+        type: string
+        description: "The unit of measure for the dataset observations"
+      uri:
+        type: string
+        description: "The uri to the location of the dataset on the web"
+  MetaDataLinks:
+    type: object
+    description: ""
+    properties:
+      access_right:
+        $ref: '#/definitions/AccessRightsLink'
+      self:
+        $ref: '#/definitions/SelfLink'
+      spatial:
+        $ref: '#/definitions/SpatialLink'
+      version:
+        $ref: '#/definitions/VersionLink'
+  VersionLink:
+    description: "The dataset version this dimension belongs too"
```

Comment:
single o on to

Reply:
also the description should probably say 'resource' rather than dimension like the properties

Reply:
👍